flake.nix

```diff
@@ -459,24 +460,22 @@
             };
           };
 
-          checks = {
-            ctl-unit-test = project.runPursTest {
-              name = "ctl-unit-test";
-              testMain = "CTL.Test.Unit";
-              env = { OGMIOS_FIXTURES = "${buildOgmiosFixtures pkgs}"; };
-            };
-            ctl-plutip-test = project.runPursTest {
-              name = "ctl-plutip-test";
-              testMain = "Test.Plutip";
-              buildInputs = [
-                (hsProjectFor system).hsPkgs.ctl-server.components.exes.ctl-server
-                pkgs.postgresql
-                pkgs.ogmios
-                pkgs.ogmios-datum-cache
-                pkgs.plutip-server
-              ];
+          checks =
+            let
+              ogmiosFixtures = buildOgmiosFixtures pkgs;
+            in
+            {
+              ctl-unit-test = project.runPursTest {
+                name = "ctl-unit-test";
+                testMain = "CTL.Test.Unit";
+                env = { OGMIOS_FIXTURES = "${ogmiosFixtures}"; };
+              };
+              ctl-plutip-test = project.runPlutipTest {
+                name = "ctl-plutip-test";
+                testMain = "Test.Plutip";
+                env = { OGMIOS_FIXTURES = "${ogmiosFixtures}"; };
```

Comment:
I think we don't need ogmios response fixtures for plutip test.

Reply:
We do, running it without having the fixtures in the env will throw an error since they're used to create the test tree